extensions/amp-analytics/0.1/visibility-manager.js

```diff
@@ -376,9 +378,27 @@ export class VisibilityManager {
           'elementHeight': layoutBox.height,
         }));
       }
+
+      // If spec includes initial scroll depth, set it
+      if (spec['initialScrollDepth']) {
```

Comment:
I think we agree to make `initialScrollDepth` and `maxScrollDepth` two visibility trigger variables by default. So there's no need to specify them in `spec`.

Reply:
Done